### ISSUE TITLE
very slow download (compared to ytd)

### ISSUE THREAD

**Issue by DrGo** [iawia002/annie#191]
Annie: 60-70  KiB/s 
ytd: 5.8Mib/s
not sure why?

For example 
➜  Downloads annie -f 250 https://www.youtube.com/watch\?v\=-7jjo8UICjQ

 Site:      YouTube youtube.com
 Title:     Most Iconic Classical Music Masterpieces Everyone Knows in One Single Video
 Type:      video
 Stream:   
     [250]  -------------------
     Quality:         audio/webm; codecs="opus"
     Size:            107.85 MiB (113084341 Bytes)
     # download with: annie -f 250 "URL"

 2.56 MiB / 107.85 MiB [>-------------------------------------]   2.38% 68.46 KiB/s 25m10

One min later using ytd
➜  inbox ytd --audio-format opus https://www.youtube.com/watch\?v\=-7jjo8UICjQ 
[youtube] -7jjo8UICjQ: Downloading webpage
[youtube] -7jjo8UICjQ: Downloading video info webpage
[download] Destination: Most Iconic Classical Music Masterpieces Everyone Knows in One Single Video--7jjo8UICjQ.f136.mp4
[download] 100% of 252.52MiB in 00:43
[download] Destination: Most Iconic Classical Music Masterpieces Everyone Knows in One Single Video--7jjo8UICjQ.f140.m4a
[download] 100% of 195.61MiB in 00:27
[ffmpeg] Merging formats into "Most Iconic Classical Music Masterpieces Everyone Knows in One Single Video--7jjo8UICjQ.mp4"
Deleting original file Most Iconic Classical Music Masterpieces Everyone Knows in One Single Video--7jjo8UICjQ.f136.mp4 (pass -k to keep)
Deleting original file Most Iconic Classical Music Masterpieces Everyone Knows in One Single Video--7jjo8UICjQ.f140.m4a (pass -k to keep)

**Comment by iawia002** [comment]
Yeah, downloading audio is really slow, I am not sure why. Did I miss some key params?

**Comment by iawia002** [comment]
It seems the reason is YouTube throttling DASH streams, and `ratebypass` seems doesn't work anymore. See https://github.com/rg3/youtube-dl/issues/15271#issuecomment-362834889

And we have to limit the chunk size to less than 10M
